shared/cmd/helpers.go

```diff
@@ -48,7 +49,7 @@ func ConfirmAction(actionText string, deniedText string) (bool, error) {
 // EnterPassword queries the user for their password through the terminal, in order to make sure it is
 // not passed in a visible way to the terminal.
 // TODO(#5749): This function is untested and should be tested.
-func EnterPassword() (string, error) {
+func EnterPassword(confirmPassword bool) (string, error) {
```

Comment:
We really should just make `confirmPassword` as default. We shouldn't even be using the non-confirm password mode

Reply:
Even for standard validator start up?

Reply:
Didnt know this is used for standard start up. I think it's fine then

Reply:
Its also used for `accounts keys`, but no password confirmation is required there.